### PR TITLE
Update mozjs to 88

### PR DESCRIPTION
**Pull request by sagudev**
https://github.com/servo/mozjs/pull/277

### CODE REVIEW

**Comment by sagudev** [comment]
Ready for merge!

File: Cargo.toml [hunk 1]
```diff
@@ -58,4 +58,4 @@ lazy_static = "1"
 libc = "0.2"
 log = "0.4"
 num-traits = "0.2"
-mozjs_sys = { git = "https://github.com/servo/mozjs", rev="95387f7235ae9ba950a8dbebb850f41709ff5b1d" }
+mozjs_sys = { git = "https://github.com/sagudev/mozjs", rev="97bfe5925c36659935a472a6b807dddebd0978ed" }
```
Comment (jdm) [inline Cargo.toml:61]: This URL needs to be updated.

Comment (sagudev) [reply Cargo.toml:61]: Yeah I forget about that.

**Comment by sagudev** [comment]
Waiting for https://github.com/servo/mozjs/pull/279.

**Comment by sagudev** [comment]
Now its ready.

**Pull request merged**